IsraelHiking.Web/src/application/services/application-initialize.service.ts

```diff
@@ -82,13 +82,13 @@ export class ApplicationInitializeService {
             if (this.runningContextService.isIFrame) {
                 return;
             }
-            if (ex.toString().indexOf("A mutation operation was attempted on a database that did not allow mutations") !== -1) {
+            if (ex.message.indexOf("A mutation operation was attempted on a database that did not allow mutations") !== -1) {
                 alert("Sorry, this site does not support running FireFox in private mode...");
             } else {
-                alert("Ooopppss... Any chance you can take a screenshot and send it to israelhikingmap@gmail.com?" +
-                    `\nInit failed: ${ex.toString()}`);
+                alert("Ooopppss... We have encountered an unexpected failure. Please try again.\nIf that does not help, please take a screenshot and send it to israelhikingmap@gmail.com?" +
```

Comment:
Lint will complain about line too long - i.e. there's a need to split this message into several lines of code, the preferred spot it the "\n".
Other than that I have no comments. Thanks!